site/en/docs/privacy-sandbox/attribution-reporting/aggregation-keys/index.md

```diff
@@ -0,0 +1,1001 @@
+---
+layout: 'layouts/doc-post.njk'
+title: 'Create aggregation keys'
+subhead: >
+  What aggregation keys are, how they're used, and how you can translate goals into keys.
+description: >
+  What aggregation keys are, how they're used, and how you can translate goals into keys.
+date: 2022-12-15
+updated: 2023-08-25
+authors:
+  - maudn
+---
+
+
+As an ad tech company running campaigns in multiple locations for various product categories, you want to help advertisers answer the following questions:
+
+1.  How many **purchases** of each product category did each of my campaigns in each geographic region generate?
+1.  How much **revenue** for each product category did each of my campaigns in each geographic region generate?
+
+While many ad tech companies encourage advertisers to configure a variety of conversion types, focusing on the most important conversions such as purchases is a good way to ensure that summary results are detailed and accurate for these important events.
+
+To do so, you'll need to think of what questions you want to answer before data is collected.
+
+## Dimensions, keys, and values
+
+To answer these questions, let's take a look at dimensions, keys, and values.
+
+### Dimensions
+
+To understand how your campaigns are generating revenue, as described here, you'll want to track the following dimensions:
+
+- Ad campaign ID
```

Comment:
From a purely style guide perspective, I'd expect all of these to be uniform, ie, 
Dimension name: definition
or
Just dimension name.

Not a hybrid version with both.